layouts/country/single.html

```diff
@@ -38,6 +38,7 @@ <h3 id="related">{{ T "_operator__list_title"}} in {{ .Title }}</h3>
                     </ul>
                 </div>
             </div>
+            {{ partial "image.html" (partial "helper/contentImage" . ) }}
```

Comment:
```suggestion
            {{ partial "image" (partial "helper/contentImage" . ) }}
```

The file extension for partials can be omitted, but it's optional and low priority.